<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Advertise</class>
 <widget class="QWidget" name="Advertise">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>887</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>562</width>
        <height>883</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="left">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="Center">
         <property name="spacing">
          <number>12</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="Banalce_advertise">
           <item>
            <widget class="QFrame" name="left_2">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="Banalce_advertise_2">
             <item>
              <widget class="QGroupBox" name="Balance">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>100</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>485</width>
                 <height>100</height>
                </size>
               </property>
               <property name="title">
                <string>Balance</string>
               </property>
               <widget class="QLineEdit" name="line_balance">
                <property name="geometry">
                 <rect>
                  <x>40</x>
                  <y>60</y>
                  <width>111</width>
                  <height>20</height>
                 </rect>
                </property>
                <property name="inputMask">
                 <string>99999999.00</string>
                </property>
                <property name="text">
                 <string>.</string>
                </property>
                <property name="maxLength">
                 <number>11</number>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
               <widget class="QLineEdit" name="line_transfer">
                <property name="geometry">
                 <rect>
                  <x>300</x>
                  <y>30</y>
                  <width>111</width>
                  <height>20</height>
                 </rect>
                </property>
                <property name="inputMask">
                 <string>99999999.\0\0</string>
                </property>
                <property name="text">
                 <string>.00</string>
                </property>
                <property name="maxLength">
                 <number>11</number>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
               <widget class="QPushButton" name="pushButton_transfer">
                <property name="geometry">
                 <rect>
                  <x>340</x>
                  <y>60</y>
                  <width>71</width>
                  <height>21</height>
                 </rect>
                </property>
                <property name="text">
                 <string>Transfer</string>
                </property>
               </widget>
               <widget class="QLabel" name="label_balance">
                <property name="geometry">
                 <rect>
                  <x>40</x>
                  <y>30</y>
                  <width>111</width>
                  <height>20</height>
                 </rect>
                </property>
                <property name="text">
                 <string>Balance (R$):</string>
                </property>
               </widget>
               <widget class="QLabel" name="label_value">
                <property name="geometry">
                 <rect>
                  <x>220</x>
                  <y>30</y>
                  <width>91</width>
                  <height>20</height>
                 </rect>
                </property>
                <property name="text">
                 <string>Value (R$):</string>
                </property>
               </widget>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="Create_Advertise">
               <property name="minimumSize">
                <size>
                 <width>450</width>
                 <height>400</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>450</width>
                 <height>400</height>
                </size>
               </property>
               <property name="title">
                <string>Create a Advertise</string>
               </property>
               <widget class="QPushButton" name="pushButton_create">
                <property name="geometry">
                 <rect>
                  <x>310</x>
                  <y>330</y>
                  <width>94</width>
                  <height>21</height>
                 </rect>
                </property>
                <property name="text">
                 <string>Create</string>
                </property>
               </widget>
               <widget class="QTextEdit" name="text_description">
                <property name="geometry">
                 <rect>
                  <x>140</x>
                  <y>170</y>
                  <width>281</width>
                  <height>121</height>
                 </rect>
                </property>
                <property name="font">
                 <font>
                  <strikeout>false</strikeout>
                 </font>
                </property>
                <property name="html">
                 <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Cantarell'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;justify&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
               <widget class="QLabel" name="label_quantity">
                <property name="geometry">
                 <rect>
                  <x>50</x>
                  <y>140</y>
                  <width>81</width>
                  <height>20</height>
                 </rect>
                </property>
                <property name="text">
                 <string>Quantity</string>
                </property>
               </widget>
               <widget class="QLabel" name="label_title">
                <property name="geometry">
                 <rect>
                  <x>50</x>
                  <y>80</y>
                  <width>68</width>
                  <height>20</height>
                 </rect>
                </property>
                <property name="text">
                 <string>Title</string>
                </property>
               </widget>
               <widget class="QLabel" name="labe_text_character">
                <property name="geometry">
                 <rect>
                  <x>170</x>
                  <y>287</y>
                  <width>131</width>
                  <height>20</height>
                 </rect>
                </property>
                <property name="font">
                 <font>
                  <pointsize>9</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Characters remaining</string>
                </property>
               </widget>
               <widget class="QComboBox" name="comboBox_category">
                <property name="geometry">
                 <rect>
                  <x>140</x>
                  <y>50</y>
                  <width>221</width>
                  <height>22</height>
                 </rect>
                </property>
                <property name="currentIndex">
                 <number>0</number>
                </property>
                <item>
                 <property name="text">
                  <string>Select a Category</string>
                 </property>
                </item>
               </widget>
               <widget class="QLabel" name="label_price">
                <property name="geometry">
                 <rect>
                  <x>50</x>
                  <y>110</y>
                  <width>68</width>
                  <height>20</height>
                 </rect>
                </property>
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Price &lt;span style=&quot; font-size:10pt; font-weight:600; font-style:italic;&quot;&gt;(R$)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
               <widget class="QLabel" name="label_n_character">
                <property name="geometry">
                 <rect>
                  <x>140</x>
                  <y>287</y>
                  <width>31</width>
                  <height>20</height>
                 </rect>
                </property>
                <property name="font">
                 <font>
                  <pointsize>9</pointsize>
                 </font>
                </property>
                <property name="layoutDirection">
                 <enum>Qt::RightToLeft</enum>
                </property>
                <property name="text">
                 <string>500</string>
                </property>
               </widget>
               <widget class="QPushButton" name="pushButton_reset">
                <property name="geometry">
                 <rect>
                  <x>150</x>
                  <y>330</y>
                  <width>94</width>
                  <height>21</height>
                 </rect>
                </property>
                <property name="text">
                 <string>Reset</string>
                </property>
               </widget>
               <widget class="QLabel" name="label_category">
                <property name="geometry">
                 <rect>
                  <x>50</x>
                  <y>50</y>
                  <width>91</width>
                  <height>20</height>
                 </rect>
                </property>
                <property name="text">
                 <string>Category</string>
                </property>
               </widget>
               <widget class="QLabel" name="label_description">
                <property name="geometry">
                 <rect>
                  <x>50</x>
                  <y>170</y>
                  <width>91</width>
                  <height>20</height>
                 </rect>
                </property>
                <property name="text">
                 <string>Description</string>
                </property>
               </widget>
               <widget class="QLineEdit" name="line_price">
                <property name="geometry">
                 <rect>
                  <x>140</x>
                  <y>110</y>
                  <width>111</width>
                  <height>20</height>
                 </rect>
                </property>
                <property name="inputMask">
                 <string>99999999.\0\0</string>
                </property>
                <property name="text">
                 <string>.00</string>
                </property>
                <property name="maxLength">
                 <number>11</number>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
               <widget class="QLineEdit" name="line_title">
                <property name="geometry">
                 <rect>
                  <x>140</x>
                  <y>80</y>
                  <width>281</width>
                  <height>20</height>
                 </rect>
                </property>
               </widget>
               <widget class="QSpinBox" name="spinBox_quantity">
                <property name="geometry">
                 <rect>
                  <x>140</x>
                  <y>140</y>
                  <width>60</width>
                  <height>22</height>
                 </rect>
                </property>
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>999</number>
                </property>
               </widget>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QFrame" name="right_2">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QFrame" name="Active_ads">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>450</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="TitleAds">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>50</height>
               </size>
              </property>
              <property name="locale">
               <locale language="English" country="UnitedStates"/>
              </property>
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; text-decoration: underline; color:#087add;&quot;&gt;Active Ads&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="Ads">
              <property name="spacing">
               <number>20</number>
              </property>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QFrame" name="right">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
